onnxruntime/contrib_ops/cuda/bert/multihead_attention.cc

```diff
@@ -153,8 +153,32 @@ Status MultiHeadAttention<T>::ComputeInternal(OpKernelContext* context) const {
       parameters.sequence_length < min_seq_len_for_flash_attention_packed_qkv_) {
     use_flash_attention = false;
   }
+  // Allocate buffers
+  size_t softmax_lse_accum_bytes = 0;
+  size_t out_accum_bytes = 0;
+  if (use_flash_attention) {
+    // split kv buffers
+    parameters.num_splits = onnxruntime::flash::num_splits_heuristic(
+        parameters.batch_size, parameters.sequence_length, parameters.kv_sequence_length, parameters.num_heads,
+        parameters.head_size, device_prop.multiProcessorCount, 128, false,
+        device_prop.major == 8 && device_prop.minor > 0);
```

Comment:
device_prop.minor >= 0?
I think sm_8x includes 80.

Reply:
This wasn't for support but rather a parameter that changed based on sm... However, you're right that they seem to have changed the heuristic as sm is not used to calculate num_splits anymore.

Reply:
I see. Shall we update num_splits_heuristic and other code as well?